src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import "./Person.scss";
+
+export const Person = ({
+  person: { name, age, sex, isMarried, partnerName },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+    {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+    {isMarried ? (
+      <p className="Person__partner">
+        {`${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`}
```

Comment:
${sex === 'm' ? 'wife' : 'husband'}`} please move this condition to separate variable, it will be cleaner